### PR TITLE
release/0.10.1

### PR DESCRIPTION
**Pull request by thefosk**
Release 0.10.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +10,16 @@
 
 ### Added
 
+- Admin API:
+  - Targets can now be deleted with
+      `DELETE /upstreams/{name}/targets/{target}`
+      [#2256](https://github.com/Mashape/kong/pull/2256)
+
```
Comment (thibaultcha) [inline CHANGELOG.md]: no blank line here

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,9 +10,16 @@
 
 ### Added
 
+- Admin API:
```
Comment (thibaultcha) [inline CHANGELOG.md]: Missing the new active targets endpoint

Comment (thefosk) [inline CHANGELOG.md]: added with last commit.

Comment (thibaultcha) [inline CHANGELOG.md]: shouldn't be part of another commit though

Comment (thefosk) [inline CHANGELOG.md]: squashed - also added a new change log about Serf.

**Comment by thibaultcha** [comment]
Manually merged to master+next with missing changelog entries and proper commit message.
